src/gridfs/file.rs

```diff
@@ -288,18 +288,18 @@ impl File {
     // Retrieves a binary file chunk and asynchronously pre-loads the next chunk.
     fn get_chunk(&mut self) -> Result<Vec<u8>> {
         let id = self.doc.id.clone();
-        let chunk_num = self.chunk_num;
+        let curr_chunk_num = self.chunk_num;
```

Comment:
Jus to make sure I understand, the lint here was to remove the shadowed variable name?